b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeAddUser.js

```diff
@@ -45,7 +60,35 @@ export default async function decodeAddUser(session, firstName, familyName, emai
                 "primary": true
             }
         ]
+    };
+}
+
+function getAddUserBody(inviteConst, firstName, familyName, email, username, password) {
+    switch (inviteConst) {
+        case InviteConst.INVITE:
+            return inviteAddUserBody(firstName, familyName, email, username);
+
+        case InviteConst.PWD:
+            return pwdAddUserBody(firstName, familyName, email, username, password);
+
+        default:
+            break;
     }
+}
+
+/**
+ * 
```

Comment:
Add a method description and add parameter descriptions. Check other applicable places as well.

Reply:
These have been fixed when this file was moved to a different location

### File
- https://github.com/wso2/samples-is/blob/master/b2b-sample/libs/business-admin-app/data-access/data-access-controller/src/lib/controller/user/controllerAddUser/controllerDecodeAddUser.ts